index.html

```diff
@@ -58,7 +58,7 @@ <h1>Junior Developer Group</h1>
         <a href="/pages/about.html">More Info</a>
       </div>
       <div class="hero_header-logo">
-        <img src="./assets/images/website-images/logo-with-background.png" />
+        <img src="./assets/images/website-images/logo-with-background.png"  alt="logo"/> 
```

Comment:
I'm not sure if you've used alt text before but 'logo' will not be an acceptable change here, unfortunately, have a look on discord - I have published instructions on how to write good alt text